Code/Engine/Foundation/Math/Color.h

```diff
@@ -246,6 +246,7 @@ class EZ_FOUNDATION_DLL ezColor
   /// \brief Sets all four RGBA components.
   void SetRGBA(float fLinearRed, float fLinearGreen, float fLinearBlue, float fLinearAlpha = 1.0f); // [tested]
 
+  void SetKelvin(ezUInt32 uKelvin);
```

Comment:
I think this should be a static make function:
static ezColor MakeFromKelvin(ezUInt32 uiKelvin);

Reply:
Oh and with a nice descriptive comment, please :)

Reply:
fixed